pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -48,6 +48,9 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 	}
 	currentVersion := currentClusterVersion.String()
 	latestVersion := kubernetes.LatestVersion().String()
+	if strings.Contains(target.Cache.OsRelease["VERSION_ID"], "15.1") && !strings.Contains(latestVersion, "1.17") {
```

Comment:
As this code will most likely not be backported to 4.2 branch, there is no point of the last part of the conditional, which should always be true.

For the first part of the conditional, I would have written a more strict equality (version = 15.1, rather than contains 15.1). However, this is not necessary. You don't need to change anything, as I doubt we'll get to a point we'll have 15.10.

Reply:
Fixed.